lib/FusionInventory/Agent/Tools/Hardware.pm

```diff
@@ -10,6 +10,7 @@ use FusionInventory::Agent::Tools;
 use FusionInventory::Agent::Tools::Network;
 use FusionInventory::Agent::Tools::SNMP;
 use FusionInventory::Agent::SNMP::Device;
+use Data::Dumper;
```

Comment:
I suppose you use that for debugging, but you should remove it as it is finally not used.

Reply:
That's right, I fix it